conan/internal/api/new/qbs_lib.py

```diff
@@ -0,0 +1,103 @@
+from conan.internal.api.new.cmake_lib import source_cpp, source_h, test_main
+
+
+conanfile_sources = '''
+import os
+
+from conan import ConanFile
+from conan.tools.qbs import Qbs
+
+
+class {{package_name}}Recipe(ConanFile):
+    name = "{{name}}"
+    version = "{{version}}"
+
+    exports_sources = "*.cpp", "*.h", "*.qbs"
+    settings = "os", "compiler", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    def build(self):
+        qbs = Qbs(self)
+        qbs.profile = ""
+        qbs_config = {"products.{{name}}.isShared": "true" if self.options.shared else "false"}
```

Comment:
This can probably be made default inside ``Qbs``, but can be done in a later PR. The best would be to aim to something as ``CMake`` with the defaults of automatically managing ``self.options.shared`` if existing.